src/main/python/janusgraph_python/core/datatypes/Circle.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class Circle(object):
+    def __init__(self, longitude, latitude, radiusInKM):
+        """
+            The actual class representing Geographical Circle.
```

Comment:
I'm not really a Python programmer, but doesn't this describe the class itself rather than the constructor and should therefore be place directly after the class header?